presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -39,6 +39,8 @@ Property Value                                     Description
                                                    :doc:`/sql/revoke` commands.
 ================================================== ============================================================
 
+Ensure that authentication has been enabled when enabling authorization checks.
```

Comment:
well, this was also intended for administrators, and not end users. It's more of a warning. I don't think they will run into errors if they enable authorization checks without enabling authentication.